cmd/skupper/skupper_podman_site.go

```diff
@@ -78,7 +79,26 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
 	if routerCreateOpts.Ingress != types.IngressNoneString {
 		// Validating ingress hosts (required as certificates must have valid hosts)
 		if len(site.IngressHosts) == 0 {
-			return fmt.Errorf("At least one ingress host is required")
+			// Get all unicast IP addresses on the machine
+			addresses, err := net.InterfaceAddrs()
+			if err != nil {
```

Comment:
Maybe before calling net.InterfaceAddrs, you could also try calling `os.Hostname()`.
If the hostname is returned, it could be used (with no error here).
But if both failed, then the error below can still be returned.

Reply:
I don't know if I understood correctly: i think sometimes hostname could be different from fqdn, soI add both hostname and any fqdn different from it. Let me know.